.github/workflows/ci.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         coverage: [ 'none' ]
```

Comment:
I don't think we need this any more.  It's merely changing a 2-row vector into a 1x2 matrix, with a dimension heading that isn't used anywhere - IIUC it sets the environment variable `INPUT_COVERAGE` as well as making available `{{matrix.coverage}}` to the script, neither of which would now be used.

Reply:
Done.